streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsProducerTest.java

```diff
@@ -1098,20 +1055,16 @@ public void shouldSwallowExceptionOnEosAbortTxInvalidEpoch() {
     }
 
     private void testSwallowExceptionOnEosAbortTx(final RuntimeException exception) {
-        mockedProducer.initTransactions();
-        mockedProducer.beginTransaction();
-        expect(mockedProducer.send(record, null)).andReturn(null);
-        mockedProducer.abortTransaction();
-        expectLastCall().andThrow(exception);
-        replay(mockedProducer);
+        doReturn(null).when(mockedProducer).send(record, null);
```

Comment:
For consistency:
```suggestion
        when(mockedProducer).send(record, null).thenReturn(null);
```

Reply:
looks like this should have been 
`when(mockedProducer.send(record, null)).thenReturn(null);` not `when(mockedProducer).send(record, null).thenReturn(null);`